crates/matrix-sdk-ui/src/timeline/event_item/content/msg_like.rs

```diff
@@ -60,4 +61,10 @@ impl MsgLikeContent {
     pub fn with_in_reply_to(&self, in_reply_to: InReplyToDetails) -> Self {
         Self { in_reply_to: Some(in_reply_to), ..self.clone() }
     }
+
+    /// If `kind` is of the [`MsgLikeKind`][MsgLikeKind::Message] variant,
+    /// return the inner [`Message`].
+    pub fn as_message(&self) -> Option<Message> {
+        as_variant!(self.kind.clone(), MsgLikeKind::Message(message) => message)
```

Comment:
nit: move the `.clone()` to the right-hand side, so that we don't clone when `self` isn't a message